mentors/python/s.json

```
@@ -159,4 +159,12 @@
     "avatar_url": null,
     "bio": "I automate boring stuff using Python."
   }
```

```suggestion
  },
```
Need to add a comma here.

---

Thanks, done.